src/main/scala/sample/stream/GroupLogFile.scala

```diff
@@ -29,16 +31,13 @@ object GroupLogFile {
         case other                  => "OTHER"
       }.
       // write lines of each group to a separate file
-      runForeach {
+      mapAsync(parallelism = 4, {
```

Comment:
explain the choice of parallelism. It should be 5 btw, since there are 5 groups that needs to be consumed